src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -93,6 +93,14 @@ def create(self, size, isolate_nspace, pool, mode, uid, gid):
 
         subvol_path = os.path.join(self.base_path, str(uuid.uuid4()).encode('utf-8'))
         try:
+            # create group directory with default mode (0o755)
+            try:
+                self.fs.stat(self.group.path)
+            except cephfs.Error as e:
+                if e.args[0] == errno.ENOENT:
+                    self.fs.mkdirs(self.group.path, self.vol_spec.DEFAULT_MODE)
+                else:
+                    raise VolumeException(-e.args[0], e.args[1])
```

Comment:
Just a stray comment from a visitor to this pr, but since this code is basically repeated 3 times why not create a utility function for this?

Reply:
Done